account_invoice_refund_link/models/account_invoice.py

```diff
@@ -9,7 +9,7 @@
 class AccountInvoice(models.Model):
     _inherit = 'account.invoice'
 
-    refund_reason = fields.Text(string="Refund reason")
+    refund_reason = fields.Text("Refund reason")
```

Comment:
Don't remove keyword attribute. They are always more flexible and it's already there. Why increasing the diff?